manifests/config.pp

```diff
@@ -38,6 +38,30 @@
     mode    => '0640',
   }
 
+  service { 'supervisord':
```

Comment:
how about putting this in a separate file and including it based on a variable in init.pp?

Reply:
So it doesn't happen on production yet